Makefile

```diff
@@ -24,10 +24,15 @@ env:
 	virtualenv --python=python3 env
 	# Install utils
 	ln -sf $(PWD)/utils env/lib/python3.*/site-packages/
+	# Install prjuray-tools
+	ln -sf $(PWD)/third_party/prjuray-tools/prjuray env/lib/python3.*/site-packages/
```

Comment:
This should be added to `requirements.txt`, which will on the next line will be installed in setup mode with a sym link.

Reply:
Ok. But for that I have to add `setup.py` to `prjuray-tools/prjuray`.

Reply:
I've opened a PR with that: https://github.com/SymbiFlow/prjuray-tools/pull/6

Reply:
The PR had been merged.

Reply:
This line should be removed now?